src/utils/stats_processor.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .opt_in_checker import OptInChecker
+import logging as log
+import os
+import json
+
+
```

Comment:
can you please document what tool will use stats file/ describe scenario please?

Reply:
Done.